### PR TITLE
Make the default inventories named inventory.ini

### PR DESCRIPTION
**Pull request by dagwieers**
So I like the fact that the default inventory for the demo was not named just *hosts* but *inventory* instead because that is confusing in normal text (easily overlooked that it is a file), but also because it doesn't just include hosts, also groups and variables.

So the thing I would definitely change is adding the extension so it stands out even more as a file, but also the content is clear. And you may get (some) syntax highlighting out of it, although it's not using an actual INI file format...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,7 @@ extra_vars.yml
 instructor*
 .vagrant/*
 ansible.cfg
-hosts
```
Comment (tima) [inline .gitignore]: Despite changing the name, I'd just assume leave this line in for backward compatibility issues.

Comment (dagwieers) [reply .gitignore]: I don't mind leaving it in, but I also don't see the point.

Comment (dfederlein) [reply .gitignore]: Not all inventory files are ini format, and we address this in the documentation here: https://docs.ansible.com/ansible/latest/intro_inventory.html

For consistency I'd like to keep hosts.

**Comment by dfederlein** [comment]
BTW, in case it hasn't been made clear, @dagwieers I am very grateful for the help, as I am sure @tima is also.

**Comment by dagwieers** [comment]
@liquidat I agree fully. I think we should start using clearer names everywhere in the documentation as well.

**Comment by tima** [comment]
I get what you are suggesting and don’t necessarily disagree @dagwieers however consistency is a primary objective of lightbulb even if that means being consistently wrong ;) (Filenames like site.yml and main.yml have never quite sat well with me personally FWIW.)

Changing this convention needs to be discussed with the greater Ansible project leaders such as @thaumos and the docs team.

**Comment by dagwieers** [comment]
@tima I understand that harmonizing this with the Ansible documentation is a great goal, but I think it is worthwhile in itself to make Ansible Lightbulb easier to understand by making good naming choices in files.

Ansible leaves it up to the user to name their inventories, so I don't see the real issue with picking a better name altogether.

PS Ansible will always have the issue with the default */etc/ansible/hosts* so do we want to drag this legacy with us for all other inventories, or not ?

File: .gitignore (hunk 1 above)
Comment (dagwieers) [reply .gitignore]: I am confused, you like to keep *hosts* in *.gitignore*, even when it is not being used within this project ?

**Comment by dfederlein** [comment]
So if we need to change the docs to be clearer and get rid of some legacy cruft, I am all for that.

Should we change it here, then change it in docs also?

**Pull request merged**